opencensus/trace/internal/span.cc

```diff
@@ -202,6 +202,12 @@ void Span::SetStatus(StatusCode canonical_code,
   }
 }
 
+void Span::SetName(absl::string_view message) const {
```

Comment:
Why called message?

Reply:
Oops, not sure what i was thinking. Corrected.